libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F412Rx.py

```diff
@@ -44,7 +44,7 @@
 	"QUADSPI" 	:	[(2,7,3)],
 	"SDIO"    	:	[(2,3,4),(2,6,4)],
 	"SPI1_RX" 	:	[(2,0,3),(2,2,3)],
-	"SPI1_TX" 	:	[(2,2,2),(2,3,3),(2,5,3)],
+	"SPI1_TX" 	:	[(2,3,3),(2,5,3)],
```

Comment:
why change the SPI1 DMA map? was it incorrect

Reply:
There is no error in viewing DMA on STM32CubeMX. You said that it does not work in https://github.com/ArduPilot/ardupilot/pull/12735/commits/3233f3b371029b5237d5b882738e6aa8b78941db. after test, cancel the  (2, 2, 2)  then it can run normally